PendingReleaseNotes

```diff
@@ -23,3 +23,6 @@
   as net/ceph. Future development releases will be available via net/ceph-devel
 
   More details about this port are in: README.FreeBSD
+
+* The inline lz4 compression plugin is now built by default and ships with
```

Comment:
might want to update `doc/rados/configuration/bluestore-config-ref.rst` accordingly. (search for `lz4` in that file).